website/docs/reference/widgets/table/inline-editing.md

```diff
@@ -235,13 +235,117 @@ This property contains the details of the row that triggered the action (`onSubm
 }
 ```
 
-## Events
+### Events
 
 | Events        | Description                                                                                                            |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------- |
 | **onSubmit**  | Action that gets triggered when the user moves away from editing a cell by pressing the enter key or clicking outside. |
-| **onSave**    | Action that gets triggered when the user clicks the save button.                                                       |
-| **onDiscard** | Action that gets triggered when user clicks discard button                                                             |
+| **onSave**    | Action that gets triggered when the user clicks the save button for a cell or a new row.                                                       |
+| **onDiscard** | Action that gets triggered when user clicks discard button for a cell or a new row.                                                       |
+
+## Adding new rows
+
+In addition to editing individual cells, you may allow users to use the Table UI to submit new rows of data. This works similarly to the Inline Editing features.
+
+### How to add a new row
+
+1. Select your Table widget, and ensure that at least one of its columns has its **Editable** property turned on.
+  - Even in new rows, only **Editable** cells can accept user input.
+2. Next, find the **Allow adding a row** property and turn it on as well.
+3. A new button should appear in the Table's UI called 'Add new row'. Click this button to add a row of new cells to the top of the table.
```

Comment:
Table's UI sounds vague. I think this should be 

"A button named **Add new row** will appear in the Table header. Click this button to add a new row"

Reply:
@jnikhila I probably wouldn't prefer "Add a new empty row" only because of a tiny technicality -- If the Table's **Default Value** property has been filled, then the new row will not be empty when it is created.

Or if you like, I could change the bit to the above, and add a little `info` block or something colorful to let users know how to fill the **Default Value** bit if they don't want the new row to be empty!

(Also, Dancia's comment has been addressed)

Reply:
@spark-c - It's getting a bit confusing when you introduced default value. May be you could rewrite it to something that says `adding a new row or a row added with default values if default values are set`. Adding an info is not required. You could also check if you could link to an existing doc that explains Default Value.

Reply:
Sure, I've updated the sentence a bit to hopefully be more straightforward (removed the nested bullet point) and I also added a link to the sections above on the page where the doc describes **Default Values**